### PR TITLE
Test localisation merge

### PR DESCRIPTION
**Pull request by ice-blaze**

### CODE REVIEW

File: resources/views/project_comments/private_index.blade.php [hunk 1]
```diff
@@ -1,31 +1,31 @@
 @extends('layouts.layout')
 
 @section('content')
-    <div class="row">
```
Comment (ice-blaze) [inline resources/views/project_comments/private_index.blade.php]: ech indentation

File: resources/views/projects/index.blade.php [hunk 2]
```diff
@@ -2,100 +2,100 @@
 
 @section('content')
 
-    @if(Auth::user())
```
Comment (ice-blaze) [inline resources/views/projects/index.blade.php]: ech identation

File: resources/views/welcome.blade.php [hunk 3]
```diff
@@ -1,26 +1,24 @@
 @extends('layouts.layout')
 
 @section('content')
-    <div class="container-fluid">
```
Comment (ice-blaze) [inline resources/views/welcome.blade.php]: ech indentation

File: resources/views/project_comments/private_index.blade.php [hunk 4]
```diff
@@ -2,29 +2,29 @@
...
 @section('content')
     <div class="row">
-        <h1>Private chat of project: <a href="{{$project->path()}}">{{$project->name}}</a></h1>
+        <h1>{{ Trans('project_comments.private_chat_of_project') }}: <a href="{{$project->path()}}">{{$project->name}}</a></h1>
     </div>
     <div class="row">
 
         {{-- use template ? --}}
         @foreach($project->privateComments as $comment)
-            <li class="list-group-item">
```
Comment (ice-blaze) [inline resources/views/project_comments/private_index.blade.php]: ech indentation

**Review by MichaelCaraccio — DISMISSED** [review]

File: resources/views/projects/index.blade.php [hunk 5]
```diff
@@ -49,51 +49,51 @@
             <div class="card-deck">
                 @foreach($threeProjects as $project)
                     <div class="card">
-                          <a href="{{ $project->path() }}">
```
Comment (ice-blaze) [inline resources/views/projects/index.blade.php:52]: Now it's correct. Was wrong on the develop branch

**Review by MichaelCaraccio — APPROVED** [review]

**Pull request merged**